llvm/lib/Transforms/InstCombine/InstCombineShifts.cpp

```diff
@@ -368,12 +368,11 @@ static Instruction *foldShiftOfShiftedBinOp(BinaryOperator &I,
 
   // Find a matching one-use shift by constant. The fold is not valid if the sum
```

Comment:
The "one-use" part of this comment is no longer completely accurate.

Reply:
Fixed!